src/components/vm/vmMigrateDialog.jsx

```diff
@@ -0,0 +1,205 @@
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+import React, { useState } from 'react';
+import {
+    Alert,
+    Button,
+    Checkbox,
+    Form,
+    FormGroup,
+    Modal,
+    Radio,
+    TextInput,
+    Tooltip
+} from '@patternfly/react-core';
+import { OutlinedQuestionCircleIcon } from "@patternfly/react-icons";
+
+import { migrateToUri } from '../../libvirt-dbus.js';
+import { isEmpty, isObjectEmpty } from '../../helpers.js';
+import { ModalError } from 'cockpit-components-inline-notification.jsx';
+
+const _ = cockpit.gettext;
+
+const DestUriRow = ({ validationFailed, destUri, setDestUri }) => {
+    return (
+        <FormGroup label={_("Destination URI")} fieldId="dest-uri"
+                   id="dest-uri"
+                   validated={validationFailed.destUri ? "error" : "default"}>
+            <TextInput id='dest-uri-input'
+                       validated={validationFailed.destUri ? "error" : "default"}
+                       value={destUri}
+                       placeholder={cockpit.format(_("Example, $0"), "qemu+ssh://192.0.2.16/system")}
+                       onChange={setDestUri} />
+        </FormGroup>
+    );
+};
+
+const OptionsRow = ({ temporary, live, setTemporary, setLive, setStorage }) => {
+    return (
+        <FormGroup label={_("Options")}
+                   fieldId="options"
+                   id="options"
+                   hasNoPaddingTop>
+            <Checkbox id="live"
+                      isChecked={live}
+                      label={_("Run after migration")}
+                      onChange={b => {
+                          setLive(b);
+                          if (!b)
+                              setStorage("shared");
+                      }} />
+            {live && <Checkbox id="temporary"
+                               isChecked={temporary}
+                               label={
+                                   <>
+                                       {_("Move temporarily")}
+                                       <Tooltip
+                                           position="top"
+                                           content={
+                                               <>
+                                                   <p>{_("When shut off on the destination, virtual machine will return to the original host in a shut off state.")}</p>
+                                                   <p>{_("This feature is often used for hardware maintanance or load-balacing.")}</p>
```

Comment:
I don't think we are interested in this comment here. I don't think such suggetions belong to the inline documentation. @skobyda was this somehow communicated witg @garrett ?

Reply:
Yes it comes out of mockup from here: https://github.com/cockpit-project/cockpit/pull/15407#issuecomment-804078472

Reply:
Is "Move temporarily" enough for this to stand on its own? We either need to have a label that communicates what this does by itself or explain it enough in a tooltip.

That said, the tip text is quite long (sorry). We could shorten it and/or improve the label. Or we could outright drop the text if you think label is fine enough.

Reply:
I don't think he label is enough. And unfortunately this option requires a lot of excplanation, we need to keep the tooltip long. So I 'd rather copy the same explanation they have in virt-manager, and rethink it in the future.

Their version is really descriptive of the option, and although long, it gives the user all the information they should have.